google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/appengine/cloud/flexible/AppEngineFlexibleStage.java

```diff
@@ -120,9 +119,20 @@ public void stage(@NotNull Path stagingDirectory) {
       if (isCustomRuntime) {
         FileUtils.copyDirectory(Paths.get(dockerDirectory).toFile(), stagingDirectory.toFile());
       }
-    } catch (IOException | InvalidPathException | MalformedYamlFileException ex) {
-      loggingHandler.print(ex.getMessage() + "\n");
-      throw new RuntimeException(ex);
+    } catch (Exception e) {
```

Comment:
I took them out because they weren't handled individually in any way, but this would also catch runtime exceptions we probably don't want to be caught so I'll change it back.